src/instructlab/lab.py

```diff
@@ -828,6 +855,12 @@ def convert(self, value, param, ctx) -> "torch.device":
         "(reduces GPU VRAM usage and may slow down training)"
     ),
 )
+@click.option(
+    "--model-name",
+    default="instructlab/merlinite-7b-lab",
+    show_default=True,
+    help="model name to use in training",
```

Comment:
chore: Might want to change this to something like  "huggingface path to use in training" just so users don't only put only `merlinite-7b-lab` instead of `instructlab/merlinite-7b-lab`.

Reply:
Can we extrapolate any of this data from other args? If not let's update the help text so it's less confusing for users